apps/docs/components/collapsible/Collapsible.tsx

```diff
@@ -1,54 +1,31 @@
 "use client";
 
 import clsx from "clsx";
-import { type ReactNode, useEffect, useRef, useState } from "react";
-import { ToggleButton } from "react-aria-components";
+import type { ReactNode } from "react";
+import { Button, UNSTABLE_Disclosure as Disclosure, UNSTABLE_DisclosurePanel as DisclosurePanel } from "react-aria-components";
 import { Icon, CollapseIcon } from "@/components/icon";
 
 import "./collapsible.css";
 
 export interface CollapsibleProps {
```

Comment:
Should we not extend the Disclosure properties?

Reply:
we might as well, then you probably don't need to add most of the props yourself.

Reply:
Exactly